quipucords/api/credential/serializer.py

```diff
@@ -198,9 +200,25 @@ def validate_satellite_cred(self, attrs):
         )
         return attrs
 
+    def validate_openshift_cred(self, attrs):
+        """Validate the attributes for openshift credentials."""
+        # Required field for OpenShift credential
+        if not self.partial:
+            auth_token = attrs.get("auth_token")
+
+        if not auth_token:
```

Comment:
if partial is `False`, the variable `auth_token` will not be attributed.

Reply:
Do we want to allow the user to update the auth_token as well ?

Reply:
Good reminder, thanks!

Reply:
if you create a variable inside a if/else and will use it outside the nested scope, you need to create it in other conditions as well (or outside the if/else scope).